src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResultSet.java

```diff
@@ -2398,6 +2398,13 @@ public Object getObject(int columnIndex) throws SQLServerException {
                     returnValue = ldt.toLocalTime();
                 }
             }
+        } else if (type == java.time.OffsetDateTime.class) {
+            microsoft.sql.DateTimeOffset dateTimeOffset = getDateTimeOffset(columnIndex);
+            if (dateTimeOffset == null) {
+                returnValue = null;
+            } else {
+                returnValue = dateTimeOffset.getOffsetDateTime();
+            }
         } else if (type == microsoft.sql.DateTimeOffset.class) {
```

Comment:
Would you add the same support for `java.time.OffsetTime`?

Reply:
Thanks for the review, @ulvii !
I'll look into it.

Reply:
I have updated the PR.
Let me know if there is anything you want me to change. :)